external-deps/spyder-kernels/spyder_kernels/customize/tests/test_umr.py

```diff
@@ -74,7 +74,7 @@ def test_umr_run(user_module):
 
     from foo1.bar import square
     umr.run()
-    umr.modnames_to_reload == ['foo', 'foo.bar']
+    assert umr.modnames_to_reload == ['foo', 'foo.bar']
```

Comment:
Please revert this change and open a PR with it in [spyder-kernels](https://github.com/spyder-ide/spyder-kernels) with it. That's because this is a copy of that repo that we include to make contributors life easier.